### PR TITLE
some clean-up 

### PR DESCRIPTION
**Pull request by DakJaniels**

### CODE REVIEW

File: extensions/units.lua [hunk 1]
```diff
@@ -10,6 +10,7 @@ HodorReflexes.units = {}
 HR_EVENT_UNITS_UPDATED = "UnitsUpdated"
 
 local HR = HodorReflexes
+local M = HR.modules.share
```
Comment (m00nyONE) [inline extensions/units.lua:13]: share module is not needed here.
M describes only the current module of the file and not allways "share"

File: modules/share/main.lua [hunk 2]
```diff
@@ -286,14 +286,14 @@ end
 -- Send player's ultimate and damage via map ping.
 -- This function must always be called from SendAttempt(), which checks whether it's safe to send data.
 local function SendData()
-	local pingType, ultType, ult, dmg, dps = DAMAGE_UNKNOWN, ULT_HORN, 0, 0, 0
+	local pingType, ultType, ult, dmg, dps, lastPingTime = DAMAGE_UNKNOWN, ULT_HORN, 0, 0, 0, GetGameTimeMilliseconds()
```
Comment (m00nyONE) [inline modules/share/main.lua:289]: lastPingTime is handled by LibDataShare when entry is nil. But its okay to include it here for consistency

File: modules/share/main.lua [hunk 3]
```diff
@@ -286,14 +286,14 @@ end
...
 	local shareHorn = SV.enableUltimateShare ~= 0 and (hornSlotted and SV.enableUltimateShare or hasSaxhleel and type(SV.enableUltimateShare) == 'number' and SV.enableUltimateShare > 0)
 	local shareColos = SV.enableColosShare and colosSlotted
 	local shareAtronach = SV.enableAtronachShare and atronachSlotted
 	local shareMiscUltimates = SV.enableMiscUltimateShare
 	if M.IsEnabled() then
 		ultType = ULT_MISC
-		ult = roundDownToFive(zo_min(500, GetUnitPower("player", POWERTYPE_ULTIMATE)))
+		ult = roundDownToFive(zo_min(500, GetUnitPower("player", COMBAT_MECHANIC_FLAGS_ULTIMATE)))
```
Comment (m00nyONE) [inline modules/share/main.lua:296]: where does this flag come from? Where there API changes? i can not find it here: https://sir.insidi.at/or/apiexplorer/#q=COMBAT_MECHANIC_FLAGS_ULTIMATE

is the APIExplorer outdated ?

File: modules/share/main.lua [hunk 4]
```diff
@@ -1230,7 +1230,7 @@ function M.IsUnitNearby(tag)
 			if IsUnitInCombat('player') and DoesUnitExist('boss1') then
 				-- during boss fights everyone is ported to the same room
 				-- but on the last boss we need to exclude portal group
-				local _, hp = GetUnitPower('boss1', POWERTYPE_HEALTH)
+				local _, hp = GetUnitPower('boss1', COMBAT_MECHANIC_FLAGS_HEALTH)
```
Comment (m00nyONE) [inline modules/share/main.lua:1233]: same here https://sir.insidi.at/or/apiexplorer/#q=COMBAT_MECHANIC_FLAGS_HEALTH

File: modules/share/main.lua [hunk 5]
```diff
@@ -2021,4 +2021,4 @@ end
 --SLASH_COMMANDS["/hrscd"] = function(str)
 --	share:SendData(tonumber(str))
 --end
-RuESO_doubleNamesNPC = nil -- disable RuESO double names, because it breaks CMX boss detection
\ No newline at end of file
+RuESO_doubleNamesNPC = nil -- disable RuESO double names, because it breaks CMX boss detection
```
Comment (m00nyONE) [inline modules/share/main.lua:2024]: maybe it's just git, but i can not see a change here

File: modules/vote/main.lua [hunk 6]
```diff
@@ -149,7 +149,7 @@ local function GroupElectionResult(eventCode, electionResult, descriptor)
 					if IsUnitOnline(group_tag) then
 						CHAT_ROUTER:AddSystemMessage(strformat('|cFF0000%s %s|r', units.GetDisplayName(group_tag), GetString(HR_VOTE_NOT_READY_CHAT)))
 					else
-						CHAT_ROUTER:AddSystemMessage(zo_strformat("<<1>><<2>> (OFFLINE)", color_orange, GetUnitDisplayName(group_tag)))
+						CHAT_ROUTER:AddSystemMessage(zo_strformat("<<1>><<2>> (OFFLINE)", ZO_ORANGE, GetUnitDisplayName(group_tag)))
```
Comment (m00nyONE) [inline modules/vote/main.lua:152]: that will crash the script. ZO_ORANGE is a table and not a string. Please remove this line here. i already fixed it here: https://github.com/m00nyONE/HodorReflexes/commit/70d4b03a7cac925c5279a1f237d354cbc5e0250d

**Review by m00nyONE — CHANGES_REQUESTED** [review]
first of all thanks a lot for your contribution :-) !
please read my review and make some changes to the code so it can be merged :-)

File: modules/vote/main.lua (hunk 6 above)
Comment (DakJaniels) [reply modules/vote/main.lua:152]: Sorry that was hasty, the right way to use a ZO_ColorDef is like how they use it here,

https://github.com/esoui/esoui/blob/8ff2e6779115b1342912883bbdd6db1e604b4cd5/esoui/common/globals/platformstoredialogs.lua#L142

**Comment by DakJaniels** [comment]
I'll redo the pr from git it self. Looks like my editor messed up the line endings haha sorry about that

**Pull request closed without merging**